### PR TITLE
Ignore index attribute in `CutOperation`

### PR DESCRIPTION
**Pull request by KathiaBarahona**
For plotting a hist, we need to preserve the index (contains the label of the numerical columns).

### CODE REVIEW

File: bach/bach/operations/cut.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class CutOperation:
     series: a numerical series to be segmented into bins
     bins: number of equal-width bins the series will be divided into.
     right: indicates if bin ranges should include the rightmost edge (lower bound).
+    ignore_index: if true, original index will be dropped
```
Comment (jansentom) [inline bach/bach/operations/cut.py:20]: shouldn't this be called drop_index in line with reset_index(..)?

**Review by jansentom — APPROVED** [review]
Looks good, but I left a question of variable naming.

File: bach/bach/operations/cut.py (hunk 1 above)
Comment (KathiaBarahona) [reply bach/bach/operations/cut.py:20]: mmm it's actually in line with append(..) https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.append.html

Comment (jansentom) [reply bach/bach/operations/cut.py:20]: okay, since pandas can't make up its mind, okay for me to stick to this too.

**Review by jansentom — APPROVED** [review]

**Pull request merged**